R/data_duplicated.R

```diff
@@ -0,0 +1,39 @@
+#' @title Extract all duplicates
+#'
+#' @description Extract all duplicates, for visual inspection.
+#' Note that it also contains the first occurrence of future
+#' duplicates, unlike [duplicated()] or [dplyr::distinct()]). Also
+#' contains an additional column reporting the number of missing
```

Comment:
Can you add backticks to the code functions you're linking here?

Reply:
I think that's not necessary because roxygen2 [now converts](https://stackoverflow.com/a/59336060/9370662) this notation to code-style writing and also automatically links to the other the functions documentation ([example result](https://rempsyc.remi-theriault.com/reference/extract_duplicates.html)).